### PR TITLE
OSDOCS-12754:Restructuring PSC creating a cluster guide to overview one

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-12754

Link to docs preview:
[Private Service Connect overview](https://85847--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-psc-enabled-private-cluster.html)

[OpenShift Dedicated on Google Cloud Platform (GCP) architecture models](https://85847--ocpdocs-pr.netlify.app/openshift-dedicated/latest/architecture/osd-architecture-models-gcp)

Peer review:
- [ ] Peer reviewer has approved this change.

**QE approval is not required for these changes.**

Additional information:

### CODE REVIEW

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done
/lgtm

**Comment by mletalie** [comment]
Hi @AedinC,
    Just added a couple of comments/suggestions that could be addressed at a later time once my changes in the restructuring goes live and we deal with any conflicts. 
Otherwise lgtm!

File: modules/private-service-connect-prereqs.adoc [hunk 1]
```diff
@@ -29,6 +29,11 @@ In addition to the requirements listed above, clusters configured with the **Ser
 
 For more information about the prerequisites that must be completed before deploying an {product-title} on {GCP}, see _Additional resources_.
 
+[NOTE]
+====
+PSC is supported with the Customer Cloud Subscription (CCS) infrastructure type only. To create an {product-title} on {GCP} using PSC, see _Creating a cluster on GCP with Workload Identity Federation_.
```
[the remark this answers is not in the document]
Comment (AedinC) [reply modules/private-service-connect-prereqs.adoc:34]: Yes, I'm going to change those links now- was waiting until the review was finished to make all the review changes in one go.

File: architecture/osd-architecture-models-gcp.adoc
Comment (mletalie) [inline architecture/osd-architecture-models-gcp.adoc:1]: Maybe move prereqs above PSC architecture to be more in line with where prereqs are in other assemblies?

File: osd_install_access_delete_cluster/creating-a-gcp-psc-enabled-private-cluster.adoc [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-[id="additional-resources_{context}"]
+[id="additional-resources-psc_{context}"]
 == Additional resources
-For information on {product-title} on {GCP} cluster prerequisites, see xref:../osd_planning/gcp-ccs.adoc#ccs-gcp-customer-requirements_gcp-ccs[Customer Requirements].
+* For information about {product-title} on {GCP} cluster prerequisites, see xref:../osd_planning/gcp-ccs.adoc#ccs-gcp-customer-requirements_gcp-ccs[Customer Requirements].
+
+* For information about configuring your firewalls, see xref:../osd_planning/gcp-ccs.adoc#osd-gcp-psc-firewall-prerequisites_gcp-ccs[GCP firewall prerequisites].
 
-For information about configuring your firewalls , see xref:../osd_planning/gcp-ccs.adoc#osd-gcp-psc-firewall-prerequisites_gcp-ccs[GCP firewall prerequisites].
+* For information about creating an {product-title} on {GCP} using PSC and Workload Identity Federation, see
```
Comment (mletalie) [inline osd_install_access_delete_cluster/creating-a-gcp-psc-enabled-private-cluster.adoc:20]: Does this need to be say ..."using PSC with Workload Identity Federation..." maybe?

Comment (mletalie) [inline osd_install_access_delete_cluster/creating-a-gcp-psc-enabled-private-cluster.adoc:20]:
```suggestion
* For information about creating an {product-title} on {GCP} using PSC with the Workload Identity Federation authentication type, see
```

**Pull request merged**